### PR TITLE
[ENHANCEMENT] Add map name

### PR DESCRIPTION
**Pull request by iboss21**
small config to rename the map when you click the P or ESC button

### CODE REVIEW

**Review by amantu-qbit — APPROVED** [review]

**Review by amantu-qbit — COMMENTED** [review]
Change:
Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), key, value)
to:
InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), key, value)

**Comment by izMystic** [comment]
All you need is this
```lua
CreateThread(function()
  AddTextEntry('FE_THDR_GTA0', 'Text You Want')
end)
```

**Comment by iboss21** [comment]
> All you need is this
> 
> ```lua
> CreateThread(function()
>   AddTextEntry('FE_THDR_GTA0', 'Text You Want')
> end)
> ```
> 
> Don't need a function invoking a native when that's already a native

that's amazing then. thank you izMystic

File: client/mapname.lua [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-- mapname
+-- github https://github.com/iboss21
+-- Discord iBoss#2984
+
+function AddTextEntry(key, value)
+	Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), key, value)
+end
+
```
Comment (dexter9006) [inline client/mapname.lua:8]: as said, you only need the 3 lignes below

**Review by dexter9006 — CHANGES_REQUESTED** [review]

**Comment by iboss21** [comment]
Got it thanks

File: client/mapname.lua [hunk 2]
```diff
@@ -0,0 +1,11 @@
+-- mapname
+-- github https://github.com/iboss21
+-- Discord iBoss#2984
+
```
Comment (Z3rio) [inline client/mapname.lua:4]:
```suggestion
```
Remove self promotion

File: client/mapname.lua (hunk 1 above)
Comment (Z3rio) [inline client/mapname.lua:8]: Unneeded function, since it's anyways only used once.

File: client/mapname.lua [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+function AddTextEntry(key, value)
+	Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), key, value)
+end
+
+CreateThread(function()
+  AddTextEntry('FE_THDR_GTAO', '~y~ The Name | ~b~ OF Your Server, ~s~ RP..')
```
Comment (Z3rio) [inline client/mapname.lua:10]:
```suggestion
  Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), "FE_THDR_GTAO", Config.MapText)
```

**Review by Z3rio — CHANGES_REQUESTED** [review]
Remove unneeded stuff

File: client/mapname.lua [hunk 4]
```diff
@@ -0,0 +1,7 @@
+function AddTextEntry(key, value)
+	Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), key, value)
+end
+
```
Comment (Z3rio) [inline client/mapname.lua:4]:
```suggestion
```
Unneeded function

**Review by Z3rio — APPROVED** [review]

**Pull request merged**